<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceControlForm</class>
 <widget class="QWidget" name="DeviceControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="AddingTab">
      <attribute name="title">
       <string>Добавить устройство</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Добавить реле</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Имя</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="ReleNameLE"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Текст запроса на открытие</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="QueryLE"/>
          </item>
          <item>
           <widget class="QPushButton" name="AddButton">
            <property name="text">
             <string>Добавить реле</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="EditingTab">
      <attribute name="title">
       <string>Управление устройствами</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Список устройств</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QListView" name="DevicesList"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Имя устройства</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EditNameLE"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>ID устройства</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EditIDLE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Формат запроса</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="EditQueueLE"/>
            </item>
            <item>
             <widget class="QPushButton" name="EditButton">
              <property name="text">
               <string>Редактировать</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="OKButton">
              <property name="text">
               <string>ОК</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
